web/stores/dive-log-store.js

```diff
@@ -52,6 +55,10 @@ class DiveLogStore {
 		this.lastEntry = newLogs.length > 1 ? newLogs[newLogs.length - 1].entryTime : null;
 		this.endOfStream = (newLogs.length < 100);
 	}
+
+	entryDeleted(entryId) {
+		_.remove(this.logs, e => e.logId === entryId);
```

Comment:
Not the most efficient way to do this: `O(n)` run time. Leave it for now but revisit if you decide to add bulk operations later.